proto/firecracker.proto

```diff
@@ -19,7 +19,9 @@ message VMConfiguration {
   bool init_dockerd = 5;
   bool debug_mode = 6;
   bool enable_dockerd_tcp = 7;
+  // Guest kernel version
   string kernel_version = 8;
```

Comment:
nit: we only use wire encoding for these protos so if you wanted, you could rename this field to `guest_kernel_version` without breaking anything (fine to leave as-is though)

Reply:
Oh right, nice